solutions/devsprint-matheus-reis-7/FinanceApp/Modules/ActivityDetails/ActivityDetailsViewController.swift

```diff
@@ -6,19 +6,77 @@
 //
 
 import UIKit
+import RxCocoa
+import RxSwift
 
-class ActivityDetailsViewController: UIViewController {
+final class ActivityDetailsViewController: UIViewController {
 
-    lazy var activityDetailsView: ActivityDetailsView = {
+    // MARK: - UIView properties
+    private let activityDetailsView: ActivityDetailsViewProtocol
 
-        let activityDetailsView = ActivityDetailsView()
-        activityDetailsView.delegate = self
-        return activityDetailsView
-    }()
+    // MARK: - DataSource / UseCase dependencies
+    private let service: FinanceService
+
+    // MARK: - Reactive Properties
+    private let disposeBag: DisposeBag
+    private let activityDetailsObservable = BehaviorRelay<ActivityDetails>(
```

Comment:
How about use `activityDetails` ? As the Relay it's the final receptor from a observable, don't make much sense to name it as `activityDetailsObservable` 🙂
```suggestion
    private let activityDetails = BehaviorRelay<ActivityDetails>(
```